benchmarks/pom.xml

```diff
@@ -16,11 +16,15 @@
     <version>0.1.0-SNAPSHOT</version>
   </parent>
 
+  <properties>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
+  </properties>
+
```

Comment:
I agree that we shouldn't export Javadoc for anything but the public APIs.  I had enabled Javadoc generation for everything because we have Javadoc-style comments in many classes, and I wanted to check that they were correctly formatted in CI.  If we continue to check the formatting, I think that it would be better to just put the `maven-javadoc-plugin` section under the `dev` profile, to be more consistent with the other CI-only checks.